src/ert/gui/tools/plot/plottery/plots/histogram.py

```diff
@@ -66,32 +66,33 @@ def plotHistogram(
     categories: set[str] = set()
     max_element_count = 0
     categorical = False
+
     for ensemble, datas in ensemble_to_data_map.items():
         if datas.empty:
-            data[ensemble.name] = pd.Series(dtype="float64")
+            data[ensemble.id] = pd.Series(dtype="float64")
```

Comment:
would be nice to do type annotations on `data` (line 63)?

Reply:
Yeah maybe, but I'm just fixing a bug here.

Reply:
`data: dict[UUID, pd.Series] = {}`
or something along those lines